test/Spec.hs

```diff
@@ -71,8 +72,21 @@ main =
     -- , bitmaskSpec @Word64
     -- , bitmaskSpec @Word
     , runSpec
+    , unitTests
     ]
 
+unitTests :: TestTree
+unitTests = testGroup "Unit tests"
+  [ -- Check that https://github.com/haskell/random/issues/53 does not regress
+
+    testCase "Subnormal generation respects upper bound" $
```

Comment:
The other specs have the type in their title. My suggestion would be to have a `testGroup "(Float)"` and put the `testCase`s in there.

"respects" is a bit ambiguous here, perhaps "does not exceed" is clearer?